config/osu.php

```diff
@@ -227,6 +227,7 @@
     'user' => [
         'allow_email_login' => get_bool(env('USER_ALLOW_EMAIL_LOGIN')) ?? true,
         'allow_registration' => get_bool(env('ALLOW_REGISTRATION')) ?? true,
+        'registration_mode' => presence(env('REGISTRATION_MODE')) ?? 'client',
```

Comment:
duplicate

Reply:
so I incorrectly amended commit adding the new registration page on this branch 👀